lib/recurly/invoice.rb

```diff
@@ -115,6 +115,7 @@ def invoice_number_with_prefix
       surcharge_in_cents
       tax_details
       billing_info_uuid
+      dunning_campaign_id
```

Comment:
i think we only want dunning_campaign_id to be readable and NOT writable - do we need to indicate that differently somehow here?

Reply:
That should be handled in server side validation for the API request. No indication is needed in the client library.